lib/mindwendel/brainstormings/idea_idea_label.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Mindwendel.Brainstormings.IdeaIdeaLabel do
```

Comment:
IdeaIdeaLabel? :-D

Reply:
I decided to apply the standard naming from rails when creating a join table (m:n relationship). This helps to keep the code small.

Reply:
You could go for IdeaLabelIdea which at least gives a hint that it is not a copy paste error ;-) sounds way better than ideaidealabel?